Week-1/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+
+<head>
+  <title>Assignment Week 1</title>
+  <link rel="stylesheet" href="style.css" />
+  <link href="https://fonts.googleapis.com/css?lamily=Lato" rel="stylesheet" />
+  <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/line-awesome/1.3.0/line-awesome/css/line-awesome.min.css"
+  />
+</head>
+<body>
+  <div class="page">
+    <div class="sidebar">
+      <a href="#"><i class="las la-bars"></i></a>
+      <a href="#"><i class="las la-home"></i></a>
+      <a href="#"><i class="las la-search"></i></a>
+      <a href="#"><i class="las la-volume-up"></i></a>
+      <a href="#"><i class="las la-user"></i></a>
+      <a href="#"><i class="lab la-spotify"></i></a>
+      <a href="#"><i class="las la-cog"></i></a>
+      <a href="#"><i class="lab la-soundcloud"></i></a>
+    </div>
+    <div class="content">
```

Comment:
Layout should convert to one column on small devices which is not happening currently